src/Symfony/Component/HttpClient/Retry/RetryBackOffInterface.php

```diff
@@ -21,5 +19,5 @@ interface RetryBackOffInterface
     /**
      * Returns the time to wait in milliseconds.
      */
-    public function getDelay(int $retryCount, string $requestMethod, string $requestUrl, array $requestOptions, ResponseInterface $partialResponse, \Throwable $throwable = null): int;
+    public function getDelay(int $retryCount, string $requestMethod, string $requestUrl, array $requestOptions, int $responseStatusCode, array $responseHeaders, ?string $responseContent, \Throwable $throwable = null): int;
```

Comment:
TransportExceptionInterface here also, and we should rename all argument from $throwable to $error or $exception (hello named arguments :) )